project-base/build.xml

```diff
@@ -36,6 +36,7 @@
     <property name="path.web.scripts" value="${path.web}/assets/scripts"/>
     <property name="path.web.styles.admin" value="${path.web}/assets/admin/styles"/>
     <property name="path.web.styles.front" value="${path.web}/assets/frontend/styles"/>
+    <property name="path.yaml-standards" value="${path.bin}/yaml-standards"/>
```

Comment:
try to create the same yaml-standards target in project-base/build-dev.xml as it is in ./build.xml and that in ./build.xml it will rewrite with project-base/ relative path things